packages/app/lib/entrypoints/android/main_amazon_appstore.dart

```diff
@@ -1,8 +1,12 @@
+import 'package:scanner_mlkit/scanner_mlkit.dart';
 import 'package:smooth_app/main.dart';
 
 /// Amazon App Store version with:
 /// - Barcode decoding algorithm: ZXing
 /// - Intent to launch the review
 void main() {
-  launchSmoothApp();
+  launchSmoothApp(
+    // TODO(g123k): Replace this when ZXing is ready
```

Comment:
Amazon doesn't exclude ML Kit does it

Reply:
On Amazon devices, they don't have the Google Play Services.
But standard Android devices, may have them.

So, it's simpler to say they doesn't